samples/c/hello_nv12_input_classification_ov/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ie_add_sample(NAME hello_nv12_input_classification_ov_c
```

Comment:
Why do we need to have a separate sample for new C API? Can we change the old sample?